intro-to-tests/intro-to-pytest.md

```diff
@@ -192,6 +192,14 @@ def test_zero_division():
 | `ZeroDivisionError`       | The kind of exception we expect should be specified here                                      |
 | `1 / 0`                   | In this code block, we should do our **Act** step, or invoke the function we're testing here. |
 
+### !callout-info
+
+## Testing Raised Exceptions?
+
+This curriculum will go deeper into raising exceptions eventually! For now, awareness of syntax is our friend.
+
+### !end-callout
+
 ## Check for Understanding
 
```

Comment:
### !challenge

* type: ordering
* id: a83910a4-46c6-4fd4-98be-97d8bf814491
* title: Multiple Tests

##### !question

Assume we wrote a function is_letter which takes a string argument and should return true if the string contains one character and the character is a letter. Order the following statements into 2 tests, each of which should pass.  Please ignore the lack of proper indentation.

##### !end-question

##### !answer

1. def test_is_letter():
1.    result = is_letter("a")
1.    assert result
1. def test_is_not_letter():
1.    result = is_letter("1")
1.    assert not result

##### !end-answer

### !end-challenge